src/cli/repl.rs

```diff
@@ -207,16 +213,17 @@ impl Repl<F> {
                     let env_out = self.store.get_z_expr(&output.env, &mut zstore)?.0;
                     let cont_out = self.store.get_z_cont(&output.cont, &mut zstore)?.0;
 
-                    let proof_claim = Self::proof_claim(
+                    let claim = Self::proof_claim(
                         &mut self.store,
                         (input.expr, output.expr),
                         (input.env, output.env),
                         (cont.parts(), cont_out.parts()),
                     );
-                    let commitment = Commitment::new(F::ZERO, proof_claim, &mut self.store)?;
-                    let proof_id = &commitment.hidden.value().hex_digits();
 
-                    let proof_path = proof_path(proof_id);
+                    let claim_comm = Commitment::new(F::ZERO, claim, &mut self.store)?;
+                    let claim_hash = &claim_comm.hidden.value().hex_digits();
```

Comment:
I think `claim_com.hidden` should be the same as `claim`. If that's what's intended, please write it that way to avoid confusion or mistakes. That said, I think it's a bad pattern to throw away the type information in the claim by only taking its value. If you want a single-field-element content-addressed value, then the *commitment*'s value can be used in that way. Doing it as written here is somewhat regressive and backs off from the strong typing Lurk data provides (including first-class support for commitments where appropriate).